### PR TITLE
Add mobilenet_v3 to slim_vision_model_test.py

### PR DESCRIPTION
**Pull request by asaadaldien**
I looked into tfhub and they do have mobilenet_v3, adding the classification model here.

### CODE REVIEW

File: integrations/tensorflow/e2e/slim_vision_models/slim_vision_model_test.py [hunk 1]
```diff
@@ -58,13 +56,18 @@ def get_input_shape():
     # Default input shape.
     return (1, 224, 224, 3)
 
+def get_mode(model_name):
+  if model_name.startswith('mobilenet_v3'):
+    return 'classification/5'
+  # Classification mode; 4 - is a format of the model (SavedModel TF v2).
+  return 'classification/4'
```
Comment (phoenix-meadowlark) [inline integrations/tensorflow/e2e/slim_vision_models/slim_vision_model_test.py:63]:
```suggestion

def get_mode(model_name):
  if model_name.startswith('mobilenet_v3'):
    return 'classification/5'
  # Classification mode; 4 - is a format of the model (SavedModel TF v2).
  return 'classification/4'

```

Add two empty lines between functions for yapf.

**Review by phoenix-meadowlark — COMMENTED** [review]
Glad support for this finally propagated. Please add the model to the `BUILD` file configuration as well.

**Review by antiagainst — APPROVED** [review]
Nice. Thanks Ahmed for adding this!

**Comment by asaadaldien** [comment]
> Glad support for this finally propagated. Please add the model to the `BUILD` file configuration as well.

I didn't but CI doesn't seem to run this test!

File: integrations/tensorflow/e2e/slim_vision_models/BUILD [hunk 2]
```diff
@@ -141,6 +141,8 @@ iree_e2e_cartesian_product_test_suite(
             "mobilenet_v2_100_224",
             "mobilenet_v2_130_224",
             "mobilenet_v2_140_224",
+            # MobileNetV2
```
Comment (phoenix-meadowlark) [inline integrations/tensorflow/e2e/slim_vision_models/BUILD:144]:
```suggestion
            # MobileNetV3
```

**Review by phoenix-meadowlark — APPROVED** [review]

**Comment by phoenix-meadowlark** [comment]
> I didn't but CI doesn't seem to run this test!

It doesn't in OSS (because the slim vision tests require external data), but it does run internally. Geoffrey was mentioning fixing this, but the keras applications version of MobileNetV3 does run on the OSS CI and should cover most failure cases.

**Pull request merged**